src/DssSpell.sol

```diff
@@ -57,105 +44,192 @@ contract DssSpellAction is DssAction {
     //
     uint256 constant THREE_PT_FIVE_PCT  = 1000000001090862085746321732;
 
-    // Math
-    uint256 constant MILLION = 10**6;
-    uint256 constant WAD     = 10**18;
-    uint256 constant RAD     = 10**45;
-
     // Addresses
-    address constant RWA002_OPERATOR         = 0x2474F297214E5d96Ba4C81986A9F0e5C260f445D;
-    address constant RWA002_GEM              = 0xAAA760c2027817169D7C8DB0DC61A2fb4c19AC23;
-    address constant MCD_JOIN_RWA002_A       = 0xe72C7e90bc26c11d45dBeE736F0acf57fC5B7152;
-    address constant RWA002_A_URN            = 0x225B3da5BE762Ee52B182157E67BeA0b31968163;
-    address constant RWA002_A_INPUT_CONDUIT  = 0x2474F297214E5d96Ba4C81986A9F0e5C260f445D;
-    address constant RWA002_A_OUTPUT_CONDUIT = 0x2474F297214E5d96Ba4C81986A9F0e5C260f445D;
+    address constant MCD_DOG              = 0x135954d155898D42C90D2a57824C690e0c7BEf1B;
+    address constant MCD_END              = 0xFB4713C20CCD81c53BC55e12923d37a4f0DB3E93;
+    address constant MCD_ESM              = 0x0f8B44Df587Abe2533cE7D75cB004C127Fe1a267;
+    address constant ILK_REGISTRY         = 0xa1cB9e29f1727d8a0a6E3e0c1334A2323312A2d5;
```

Comment:
```
0xa1cB9e29f1727d8a0a6E3e0c1334A2323312A2d5
└─ ward: 0xDA0FaB0700A4389F6E6679aBAb1692B4601ce9bf
   └─ eoa: user
```
```
$ seth call 0xa1cB9e29f1727d8a0a6E3e0c1334A2323312A2d5 'vat()(address)'
0x35D1b3F3D7966A1DFe207aa4514C12a259A0492B
$ seth call 0xa1cB9e29f1727d8a0a6E3e0c1334A2323312A2d5 'dog()(address)'
0x135954d155898D42C90D2a57824C690e0c7BEf1B
$ seth call 0xa1cB9e29f1727d8a0a6E3e0c1334A2323312A2d5 'cat()(address)'
0xa5679C04fc3d9d8b0AaB1F0ab83555b301cA70Ea
$ seth call 0xa1cB9e29f1727d8a0a6E3e0c1334A2323312A2d5 'spot()(address)'
0x65C79fcB50Ca1594B025960e539eD7A9a6D434A3
```

Reply:
Pause proxy is also relied in this contract. The deployer will be denied after we can add the `pip` of RWA002 that will be created when the last spell is cast.

Reply:
`dss-ward` didn't catch that ward because the Rely event on the Ilk Registry doesn't index the relied address. This probably should be fixed, but I'm gonna work on `dss-ward` to be more robust against this.